src/interactUtils.js

```diff
@@ -24,6 +24,7 @@ import $ from 'jquery';
 import _ from 'lodash';
 import interact from 'interact';
 import 'core/mouseEvent';
+import triggerCustomEvent from '@oat-sa/tao-core-sdk/src/core/customEvent';
```

Comment:
Shouldn't the SDK dependency be specified as a production dependency in `package.json`?

Reply:
No longer needed. Thanks for comment